web_ui/frontend/components/FederationOverview.tsx

```diff
@@ -23,22 +24,38 @@ const LinkBox = ({href, text} : {href: string, text: string}) => {
     )
 }
 
+const UrlData = [
+    {key: ["Federation", "NamespaceUrl", "Value"], text: "Namespace Registry"},
+    {key: ["Federation", "DirectorUrl", "Value"], text: "Director"},
+    {key: ["Federation", "RegistryUrl", "Value"], text: "Registry"},
+    {key: ["Federation", "TopologyNamespaceUrl", "Value"], text: "Topology Namespace"},
+    {key: ["Federation", "DiscoveryUrl", "Value"], text: "Discovery"},
+    {key: ["Federation", "JwkUrl", "Value"], text: "JWK"}
+]
+
 const FederationOverview = () => {
 
-    const [config, setConfig] = useState<{ [key: string] : string | undefined} | undefined>(undefined)
+    const [config, setConfig] = useState<{text: string, url: string | undefined}[]>([])
 
     let getConfig = async () => {
         let response = await fetch("/api/v1.0/config")
         if(response.ok) {
             const responseData = await response.json() as Config
 
-            setConfig({
-                JwkUrl: (responseData?.Federation as Config)?.NamespaceUrl?.Value as undefined | string,
-                NamespaceUrl: (responseData?.Federation as Config)?.NamespaceUrl?.Value as undefined | string,
-                DirectorUrl: (responseData?.Federation as Config)?.DirectorUrl?.Value as undefined | string,
-                TopologyNamespaceUrl: (responseData?.Federation as Config)?.TopologyNamespaceUrl?.Value as undefined | string,
-                DiscoveryUrl: (responseData?.Federation as Config)?.DiscoveryUrl?.Value as undefined | string,
+            const federationUrls = UrlData.map(({key, text}) => {
+                let url = getObjectValue<string>(responseData, key)
+                if(url && !url?.startsWith("http://")) {
+                    url = "http://" + url
```

Comment:
Also, for some reason, the links I got from the panel are all broken:
(note the duplicated, yet wrong protocols)

```
http://https//topology.opensciencegrid.org/stashcache/namespaces.json
```

But this is what I saw in the config page:
```
Federation.TopologyNamespaceUrl: https://topology.opensciencegrid.org/stashcache/namespaces.json
```

which is correct